requirements.txt

```diff
@@ -5,3 +5,4 @@ scipy
 pandas
 scikit-learn
 tqdm
+xarray
```

Comment:
It's better to add `xarray` in the [test_deps](https://github.com/brainets/hoi/blob/main/setup.py#L30) rather than the general requirements.